### DISCUSSION TITLE
OCS integration

### DISCUSSION THREAD

**Discussion by herlesupreeth** [herlesupreeth/docker_open5gs#135] [Ideas]
Has anyone integrated open5gs with OCS over Gy interface?

**Comment by herlesupreeth** [comment]
@s5uishida Any insights from you?

**Reply by s5uishida** [reply]
Hi @herlesupreeth 

I know there is a Gy interface in Open5GS, sorry I don't know much about how it works.
Also, is there an open source for OCS?

For the Gy interface and each NF, I saw the following for an overview.

https://github.com/nickel0/3GPP-Overall-Architecture

**Comment by herlesupreeth** [comment]
I found an opensource OCS here - https://github.com/sigscale/ocs . Haven't gone through documentation yet.

**Comment by aseaudi** [comment]
@herlesupreeth 

I integrated OCS with Open5gs, with some modifications in Open5gs and OCS
https://github.com/aseaudi/open5gs
https://github.com/aseaudi/ocs

Working demo in below video
https://www.youtube.com/watch?v=zX7QtiSXXIw

**Comment by herlesupreeth** [comment]
Sigscale OCS has now been integrated into this repo. Please refer to the README for more information.
